[2024-05-14T07:29:44.883+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-14T07:29:44.893+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_jobs104.check_if_update manual__2024-05-14T07:29:38.068071+00:00 [queued]>
[2024-05-14T07:29:44.896+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_jobs104.check_if_update manual__2024-05-14T07:29:38.068071+00:00 [queued]>
[2024-05-14T07:29:44.897+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-14T07:29:44.904+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_if_update> on 2024-05-14 07:29:38.068071+00:00
[2024-05-14T07:29:44.906+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=413) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-14T07:29:44.907+0000] {standard_task_runner.py:63} INFO - Started process 415 to run task
[2024-05-14T07:29:44.907+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_jobs104', 'check_if_update', 'manual__2024-05-14T07:29:38.068071+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/.ipynb_checkpoints/data_pipeline_jobs104-checkpoint.py', '--cfg-path', '/tmp/tmpnyhu_jt4']
[2024-05-14T07:29:44.909+0000] {standard_task_runner.py:91} INFO - Job 48: Subtask check_if_update
[2024-05-14T07:29:44.928+0000] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline_jobs104.check_if_update manual__2024-05-14T07:29:38.068071+00:00 [running]> on host 9bf348f884a5
[2024-05-14T07:29:44.967+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline_jobs104' AIRFLOW_CTX_TASK_ID='check_if_update' AIRFLOW_CTX_EXECUTION_DATE='2024-05-14T07:29:38.068071+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-14T07:29:38.068071+00:00'
[2024-05-14T07:29:44.968+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-14T07:29:44.975+0000] {logging_mixin.py:188} INFO - check_if_update !!!
[2024-05-14T07:29:44.976+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-14T07:29:44.976+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-14T07:29:44.981+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_pipeline_jobs104, task_id=check_if_update, run_id=manual__2024-05-14T07:29:38.068071+00:00, execution_date=20240514T072938, start_date=20240514T072944, end_date=20240514T072944
[2024-05-14T07:29:45.010+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-14T07:29:45.022+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-14T07:29:45.023+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
